kobo/hub/models.py

```diff
@@ -920,6 +920,7 @@ def resubmit_task(self, user, force=False):
             "arch_name": self.arch.name,
             "channel_name": self.channel.name,
             "priority": self.priority,
+            "priority": priority if priority else self.priority,
```

Comment:
It occurs to me now that 0 might be a valid priority, but it's falsey in python, so it'd be impossible to specify that value.

Can we fix that?  Meaning it likely should be "if priority is not None".

Reply:
Fixed.